deploy-agent/deployd/common/utils.py

```diff
@@ -236,7 +236,25 @@ def get_container_health_info(commit):
                         command = ['docker', 'inspect', '-f', '{{.State.Health.Status}}', name]
                         status = subprocess.run(command, check=True, stdout=subprocess.PIPE).stdout
                         if status:
-                            result.append(f"{name}:{status.decode().strip()}")
+                            status = status.decode().strip()
+                            if status == "unhealthy" and "redeploy_when_unhealthy=enabled" in parts[2]:
+                                labels = parts[2].split(',')
+                                max_retry = REDEPLOY_RETRY_TIMES
+                                for label in labels:
+                                    if "redeploy_retry_times" in label:
+                                        max_retry = int(label.split('=')[1])
+                                retry_num = 0
+                                file_name = "/mnt/deployd/" + service
+                                if os.path.exists(file_name):
+                                    f=open(file_name,"r")
+                                    retry_num = int(f.readline())
+                                    f.close()
+                                if retry_num < max_retry:
+                                    ff=open(file_name,"w")
```

Comment:
`with open(file_name, mode="w") as file:`

Reply:
sure, thank you